config/routes.rb

```diff
@@ -0,0 +1,116 @@
+Rails.application.routes.draw do
+  get 'media/index' => 'media#index', as: 'media'
+  root 'media#index'
+
+
+patch 'books/:id/upvote' => 'books#upvote'
+patch 'movies/:id/upvote' => 'movies#upvote'
+patch 'albums/:id/upvote' => 'albums#upvote'
+
+patch 'books/:id/downvote' => 'books#downvote'
+patch 'movies/:id/downvote' => 'movies#downvote'
+patch 'albums/:id/downvote' => 'albums#downvote'
+
+Rails.application.routes.draw do
```

Comment:
While it doesn't appear to be hurting anything, it is not necessary or meaningful to have another `Rails.application.routes.draw` call inside of the outer one.